src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Circle() {
+    }
+
+    public Circle(int radius, Color color) {
+    }
```

Comment:
you should initialize properties here. fix in all similar places

Reply:
not fixed